src/track/keyutils.h

```diff
@@ -12,8 +12,6 @@
 class KeyUtils {
   public:
     enum KeyNotation {
-        // The default notation (set with setNotation).
-        DEFAULT = 0,
```

Comment:
I tried making CUSTOM = 0, but then the sort order wasn't being set up properly for that notation when Mixxx started, even when I added `m_pKeyNotation->emitValueChanged();` after https://github.com/Be-ing/mixxx/blob/3a5142f3dd5db09955320f4b938cd9e921315c8c/src/preferences/dialog/dlgprefkey.cpp#L141